src/backend/base/langflow/services/cache/service.py

```diff
@@ -355,11 +355,7 @@ async def _clear(self) -> None:
         self.cache.clear()
 
     async def upsert(self, key, value, lock: asyncio.Lock | None = None) -> None:
-        if not lock:
```

Comment:
Annotate the method with `@override` (from typing_extensions) to solve the ARG002 rule error.